src/main/kotlin/no/nav/medlemskap/domene/arbeidsforhold/Arbeidsforhold.kt

```diff
@@ -69,7 +69,17 @@ data class Arbeidsforhold(
 
          fun List<PermisjonPermittering>.totaltantallDager(): Int {
 
-            val antallDagerIHverPermisjon = this.map { ChronoUnit.DAYS.between(it.periode.fom,it.periode.tom) }
+            val antallDagerIHverPermisjon = this.map {
+                var fom = it.periode.fom
+                var tom = it.periode.tom
+                if (fom == null){
```

Comment:
Kan dette forenkles slik:
`
val fom = it.periode.fom ? LocalDate.now().minusYears(1)
val tom = it.periode.tom ? LocalDate.now().minusYears(1)
`

Er det ønskelig at vi går 1 år tilbake i tid fra i dag, eller ønsker vi å bruker kontrollperioden? Isåfall kan vi komme inn med kontrollperioden som parameter i totaltantallDager funksjonen.